app/controllers/courses_controller.rb

```diff
@@ -418,6 +418,10 @@ def update_timeslice_duration
     @course.save
   end
 
+  def update_course_calendar_view
```

Comment:
This is not a very precise name for a method that is exclusively concerned with the week-start flag.